malaria24/ona/models.py

```diff
@@ -757,6 +757,17 @@ def __unicode__(self):
         return u'%s (%s)' % (self.name, self.role)
 
 
+class InboundSMS(models.Model):
+    """
+    An SMS sent to the system.
+    """
+    sender = models.CharField(max_length=255)
+    content = models.CharField(max_length=255)
```

Comment:
Maybe this is better as a `TextField`? Since a multipart SMS can easily be bigger than 255 characters.

Reply:
Also, `content` is allowed to be `null` or blank in vumi. Not sure if we want to store both nulls and blanks, or coerce one into the other in the view.